arduino/smartcar/smartcar.ino

```diff
@@ -214,6 +215,7 @@ void turnleft()
 {
   car.setSpeed (30);
   car.setAngle (-95);
+  delay(500);
```

Comment:
Is this enough to avoid the obstacle? Also, the camera will still hang for 0.5 seconds every time this code branch is executed.